dblookupext/historyRepository.go

```diff
@@ -240,51 +235,61 @@ func (hr *historyRepository) onNotarizedMiniblock(
 		return
 	}
 
+	updateHandler := hr.createUpdateHandler(metaBlockHash, metaBlockNonce, miniblockHeader, shardOfContainingBlock)
```

Comment:
:+1: 

(alternative name: `createMiniblockMetadataUpdateHandler` - since here we are in the history repository)